src/sites/twitch-twilight/modules/player.jsx

```diff
@@ -876,6 +880,15 @@ export default class Player extends Module {
 			});
 		}
 
+		if ( window.safari ) {
+			const targetMode = video.webkitPresentationMode === 'picture-in-picture' ? 'inline' : 'picture-in-picture';
+			video.webkitSetPresentationMode(targetMode);
+
+			setTimeout(() => this.addPiPButton(inst));
```

Comment:
Does Safari have events for when the video enters / leaves PiP mode? I'd prefer to use an event rather than a timeout so that when someone closes it manually the button goes into the correct state.